vendor/github.com/profitbricks/profitbricks-sdk-go/req.go

```diff
@@ -0,0 +1,128 @@
+package profitbricks
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"github.com/profitbricks/profitbricks-sdk-go/model"
+)
+
+//FullHeader is the standard header to include with all http requests except is_patch and is_command
+const FullHeader = "application/vnd.profitbricks.resource+json"
+
+//PatchHeader is used with is_patch .
+const PatchHeader = "application/vnd.profitbricks.partial-properties+json"
+
+//CommandHeader is used with is_command
+const CommandHeader = "application/x-www-form-urlencoded"
+
+var Depth = "5"
+
+// SetDepth is used to set Depth
+func SetDepth(newdepth string) string {
+	Depth = newdepth
+	return Depth
+}
+
+// mk_url  either:
+// returns the path (if it`s a full url)
+//			 or
+//	returns	Endpoint+ path .
+func mk_url(path string) string {
+	if strings.HasPrefix(path, "http") {
+		//REMOVE AFTER TESTING
+		path := strings.Replace(path, "https://api.profitbricks.com/rest/v2", Endpoint, 1)
+		// END REMOVE
```

Comment:
This is just a comment in the SDK. The if above will replace "http" path with a default REST path.